src/components/Country.jsx

```diff
@@ -1,10 +1,30 @@
-import React from 'react'
+import 'react'
 
-const Country = () => {
-  return (
-    // TODO: Country component
-    <div>Country</div>
-  )
-}
+/**
+ * Country Component
+ * Displays a country's flag, name, population, region, and capital.
+ *
+ * Props:
+ * - country: { flag, name, population, region, capital }
+ */
+const Country = ({ country }) => {
+    const { flags, name, population, region, capital } = country;
 
-export default Country
\ No newline at end of file
+    return (
```

Comment:
Good work on destruct the props. 
You can always tell this component that it will receive 
`flags, name, population, region, capital` as props and pass those from outside instead of passing an object and destruct the object

Then you will do something like this : 

```js
<Country flags={} name={} ...
```